test/framework/src/main/java/org/elasticsearch/test/disruption/LongGCDisruption.java

```diff
@@ -122,6 +170,19 @@ public synchronized void stopDisrupting() {
             resumeThreads(suspendedThreads);
             suspendedThreads = null;
         }
+        stopBlockDetection();
```

Comment:
shall we stop it first? we might get false positive as the node threads move an acquire shared resource and trigger the block detection.